src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -152,30 +160,6 @@ class SelectClauseSegment(ansi_dialect.get_segment("SelectClauseSegment")):  # t
     )
 
 
-@bigquery_dialect.segment(replace=True)
```

Comment:
This is no longer needed because we now get the same effect by copying and appending to the new `BaseExpressionElementGrammar`.